ctk/common/src/main/kotlin/org/codice/compliance/verification/core/SamlAssertionsVerifier.kt

```diff
@@ -18,47 +18,37 @@ import org.codice.compliance.SAMLCore_2_2_4_a
 import org.codice.compliance.allChildren
 import org.codice.compliance.children
 import org.codice.compliance.utils.TestCommon.Companion.ELEMENT
-import org.codice.compliance.verification.core.internal.AssertionsVerifier
-import org.codice.compliance.verification.core.internal.ConditionsVerifier
-import org.codice.compliance.verification.core.internal.StatementVerifier
-import org.codice.compliance.verification.core.internal.SubjectVerifier
+import org.codice.compliance.verification.core.assertions.AssertionsVerifier
+import org.codice.compliance.verification.core.assertions.ConditionsVerifier
+import org.codice.compliance.verification.core.assertions.NameIdentifierVerifier
+import org.codice.compliance.verification.core.assertions.StatementVerifier
+import org.codice.compliance.verification.core.assertions.SubjectVerifier
 import org.w3c.dom.Node
 
 class SamlAssertionsVerifier(val node: Node) {
-    companion object {
-        private const val ENCRYPTED_ID = "EncryptedID"
-    }
-    /**
-     * Verify assertions against the Core Spec document
-     * 2 SAML Assertions
-     */
+
+    /** 2 SAML Assertions */
     fun verify() {
         verifyNameIdentifiers()
+        NameIdentifierVerifier(node).verify()
         AssertionsVerifier(node).verify()
         SubjectVerifier(node).verify()
         ConditionsVerifier(node).verify()
         StatementVerifier(node).verify()
     }
 
     /**
-     * Verify the Name Identifiers against the Core Spec document
      * 2.2 Name Identifiers
      * 2.2.4 Element <EncryptedID>
      */
     private fun verifyNameIdentifiers() {
```

Comment:
❗️ This verification can be removed. Covered in EncryptionVerifier.kt in an upcoming PR: https://github.com/connexta/saml-conformance/pull/92/files#diff-ea01368ee04c99da2848408cd4d34ea3R65